web/elm/src/Views/Spinner.elm

```diff
@@ -10,18 +10,32 @@ module Views.Spinner exposing (spinner)
 
 import Html exposing (Html)
 import Html.Attributes exposing (style)
+import Html.Events exposing (onMouseEnter, onMouseLeave)
+import Message.Message exposing (DomID, Message(..))
 
 
-spinner : { size : String, margin : String } -> Html msg
-spinner { size, margin } =
+spinner :
```

Comment:
Grrrrr. Our lovely straightforward spinner is getting bloatier. :( Maybe we could throw in a `customizedSpinner` version, and have `spinner` take no or almost no parameters?

(Also size and margin should be Ints.)

Reply:
a31b55742fe34b931509ef2af270c3710540c6cf -- see what you think

Reply:
LGTM! (Kinda frustrating there's like one case that prevents making `margin` a float too. :( )